control/tap-ctl-spawn.c

```diff
@@ -233,18 +233,23 @@ tap_ctl_spawn(const char *slice)
 	}
 
 	id = tap_ctl_get_child_id(readfd);
-	if (id < 0)
+	if (id < 0) {
 		EPRINTF("get_id failed, child %d err %d\n", child, errno);
+		return id;
+	}
+
+	if (tap_ctl_move_to_cgroup(id, "blkio", "vm.slice") < 0)
+		EPRINTF("failed to move tapdisk %d to blkio cgroup slice: %s; ignoring.\n", id, strerror(errno));
 
 	/* Put the tapdisk in a cgroup slice (best-effort) */
 	if (!slice) {
 #ifndef TAP_CTL_NO_DEFAULT_CGROUP_SLICE
 		/* No option specified; move it to the default slice */
-		if (tap_ctl_move_to_cgroup(id, "/") < 0)
+		if (tap_ctl_move_to_cgroup(id, "cpu", "/") < 0)
 			EPRINTF("failed to move tapdisk %d to default cgroup slice: %s; ignoring.\n", id, strerror(errno));
```

Comment:
Should these contain 'cpu' in line with the one above having 'blkio'?

Reply:
Fixed.

Reply:
LGTM